### PR TITLE
HOTFIX Advanced XSS

### PR DESCRIPTION
**Pull request by SavelevMatthew**

### CODE REVIEW

**Review by sitozzz — APPROVED** [review]

File: apps/condo/domains/common/constants/regexps.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const UPPER_CASE_ALPHANUMERIC_REGEXP = /^[A-Z0-9_]+$/
 const LETTERS_AND_NUMBERS = /[\p{L}\p{N}]/gu
 const PHONE = /^\+?\d*(\.\d*)?$/
 const PHONE_CLEAR_REGEXP = /[^+0-9]/g
-const JAVASCRIPT_URL_XSS = /^[u00-u1F]*j[\r\n\t]*a[\r\n\t]*v[\r\n\t]*a[\r\n\t]*s[\r\n\t]*c[\r\n\t]*r[\r\n\t]*i[\r\n\t]*p[\r\n\t]*t[\r\n\t]*:/i
+const JAVASCRIPT_URL_XSS = /^[u00-u1F]*j[\r\n\t\s]*a[\r\n\t\s]*v[\r\n\t\s]*a[\r\n\t\s]*s[\r\n\t\s]*c[\r\n\t\s]*r[\r\n\t\s]*i[\r\n\t\s]*p[\r\n\t\s]*t[\r\n\t\s]*:/i
```
Comment (AleX83Xpert) [inline apps/condo/domains/common/constants/regexps.js:6]: I thought that `\s` is a shorthand for `[ \t\r\n\f]`

**Review by AleX83Xpert — APPROVED** [review]

File: apps/condo/domains/common/constants/regexps.js (hunk 1 above)
Comment (SavelevMatthew) [reply apps/condo/domains/common/constants/regexps.js]: Shortened expression 👍

**Comment by SavelevMatthew** [comment]
> Best to explain how this works and why this fix will fix the vulnerability

decodeURI converts string like "%09Jav%09ascript:alert(document.cookie)" to "	Jav	ascript:alert(document.cookie)", so it can be detected by regexp

**Comment by toplenboren** [comment]
> > Best to explain how this works and why this fix will fix the vulnerability
> 
> decodeURI converts string like "%09Jav%09ascript:alert(document.cookie)" to " Jav ascript:alert(document.cookie)", so it can be detected by regexp

ok, thanks

**Pull request merged**